src/test/java/edu/hm/hafner/analysis/IssuesTest.java

```diff
@@ -0,0 +1,289 @@
+package edu.hm.hafner.analysis;
+
+import java.util.Set;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.util.NoSuchElementException;
+import static edu.hm.hafner.analysis.Assertions.IssueAssert.assertThat;
+import static edu.hm.hafner.analysis.Assertions.IssuesAssert.assertThat;
+import static org.assertj.core.api.Java6Assertions.*;
+
+/**
+ * Test class for issues.
+ *
+ * @author Tom Maier
+ * @author Johannes Arzt
+ */
+class IssuesTest {
+
+    @Test
+    void addBasicIssueTest() {
+
+        Issues issues = new Issues();
+        IssueBuilder builder = new IssueBuilder();
+        Issue firstIssue = builder.build();
+        issues.add(firstIssue);
+
+        assertThat(issues)
+                .hasSize(1)
+                .hasHighPrioritySize(0)
+                .hasLowPrioritySize(0)
+                .hasNormalPrioritySize(1)
+                .hasIssueOnIndex(firstIssue, 0)
+                .hasNumberOfFiles(1)
+                .hasToString(String.format("%d issues", issues.size()));
+
+        assertThat(issues.get(0))
+                .hasLineStart(0)
```

Comment:
das wird bereits in IssueTest getestet. Besser:
`assertThat(...).isSameAs(firstIssue)`